src/components/sidebar/ContextAwareSidebar.tsx

```diff
@@ -158,12 +164,46 @@ class ContextAwareSidebar
   constructor(props) {
     super(props);
 
+    this.state = {
+      resourceRefs: Maybe.nothing(), // KEVIN-1936 (BBB) initialize state as nothing
+    };
     this.onRemovePage = this.onRemovePage.bind(this);
     this.onPageEdit = this.onPageEdit.bind(this);
     this.onAddPage = this.onAddPage.bind(this);
     this.onToggleBranching = this.onToggleBranching.bind(this);
   }
 
+  componentDidMount() {
+    this.fetchRefs();
+
+  }
+
+  fetchRefs() {
+    const { course, resource } = this.props;
+
+//    const org = this.props.selectedOrganization;
+//    const directResources = org
+
+
+      // KEVIN-1936 see (AAA) to see what call to replicate
+      // fetch edges for reference list
+    const fetchResouceEdges = persistence.fetchEdges(course.guid, {
+      // sourceType: LegacyTypes.assessment2,
+      // destinationType: LegacyTypes.assessment2_pool,
+    }).then((edges) => {
+
+      const sources = edges.filter((edge) => {
+        return edge.destinationId.indexOf(resource.id) >= 0;
```

Comment:
Using 'indexOf' here in this manner will incorrectly identify edges whose destinationIds is the resource.id plus some additional characters.  Rework this to use a string comparison.

Reply:
Updated to use string comparison with properly formatted Ids